Packs/PrismaAccess/Integrations/PrismaSASE/PrismaSASE.yml

```
@@ -0,0 +1,1891 @@
+      type: String
+    - contextPath: PrismaSase.SecurityRule.id
+      description: Security rule ID.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.log_setting
+      description: Security rule log setting.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.name
+      description: Security rule name.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.position
+      description: Security rule position.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.service
+      description: Security rule service.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.source
+      description: Security rule source.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.source_user
+      description: Security rule source user.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.tag
+      description: Security rule tag.
+      type: String
+    - contextPath: PrismaSase.SecurityRule.to
+      description: Security rule to field (destination zone(s)).
+      type: String
+    - contextPath: PrismaSase.SecurityRule.negate_destination
+      description: Security rule negate destination.
+      type: Boolean
+  - arguments:
+    - default: false
+      description: Comma-separated list of targets (Remote Networks, Mobile Users, Service Connections).
+      isArray: false
+      name: folders
+      required: true
+    - default: false
+      description: Configuration push job description.
+      isArray: false
+      name: description
+      required: false
+    - default: false
+      description: interval for polling command. Default is 30.
+      name: interval_in_seconds
+      required: false
+    - description: Polling timeout in seconds.
+      name: polling_timeout_in_seconds
+      defaultValue: '600'
+    - default: false
+      description: For polling use.
+      isArray: false
+      name: job_id
+      required: false
+      deprecated: true
+    - default: false
+      description: For polling use.
+      isArray: false
+      name: parent_finished
+      required: false
+      deprecated: true
+    - description: Tenant services group ID. If not provided, the tsg_id integration parameter will be used as the default.
+      name: tsg_id
+    deprecated: false
+    description: Push the candidate configuration.
+    execution: false
+    name: prisma-sase-candidate-config-push
+    polling: true
+    outputs:
+    - contextPath: PrismaSase.CandidateConfig.job_id
+      description: Configuration job ID.
+      type: String
+    - contextPath: PrismaSase.CandidateConfig.result
+      description: The configuration push result.
+      type: Boolean
+  - arguments:
+    - default: false
+      description: ID of the rule to be changed.
+      isArray: false
+      name: rule_id
+      required: true
+    - auto: PREDEFINED
+      default: false
+      defaultValue: Shared
+      description: Prisma sase folder location.
```

```suggestion
      description: Prisma SASE folder location.
```
Is it the folder location or the type of folder?